static/sass/_snapcraft_release.scss

```diff
@@ -87,4 +86,19 @@
   .p-contextual-menu__link.is-indented {
     padding-left: 1rem;
   }
+
+  .p-revisions-list {
+    // TODO: instead of fighting vanilla specificity add a class name
+    input+label {
```

Comment:
anything stopping you from doing that now?

Reply:
laziness